src/error/modalError.js

```diff
@@ -41,12 +41,12 @@ export default class ModalError extends Component {
        return(
             <div>
                  <Modal show={this.state.show} onHide={this.hideModal} animation='true' centered>
-                   <Modal.Header  closeButton className = "bg-warning"> 
+                   <Modal.Header  closeButton style={{backgroundColor: "#ff8e09"}}> 
```

Comment:
It looks weird, to make all functional buttons orange color. They fulfill different functionalities.

Reply:
bg-warning is also orange but a bit different from our site's color. Or you mean to change the border color instead of background?